### PR TITLE
CI / CD initial setup

### PR DESCRIPTION
**Pull request by youaresofunny**
#### What's this PR do?
So, it brings Dockerfile that runs lint / tests and then builds sources into dist folder.
Also .travis.yml added, to automate that process on Travis-ci.org

Plenty of changes was made at serverside also (http://staging-ui.trusty.apasia.tech):
 - Docker and needed tools installation
 - Permit logins for root and passwords
 - Reverse proxy for dev builds urls
 - Page with builds for Pull Requests is available here - http://staging-ui.trusty.apasia.tech:8080/
 - It works on https://github.com/youaresofunny/dockerdelivery service instance

#### Where should the reviewer start?
Dockerfile

#### How should this be manually tested?
docker build -t testbuild .
docker run --rm testbuild

All builds results can be found at https://travis-ci.org/bitshares/bitshares-community-ui

#### What are the relevant tickets?
https://github.com/bitshares/bitshares-community-ui/issues/8

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM node:latest
+WORKDIR /usr/src/app/
+COPY package.json /usr/src/app/package.json
+COPY vuex-bitshares/package.json vuex-bitshares/package.json
+RUN npm install
```
Comment (roma219) [inline Dockerfile:5]: Should run `npm run svg` here?

Comment (youaresofunny) [reply Dockerfile:5]: Yes, sure

**Review by roma219 — APPROVED** [review]
👍

**Comment by hisinbox** [comment]
cornerstone feature and very well documented

**Pull request merged**